psydac/api/tests/test_2d_navier_stokes.py

```diff
@@ -362,22 +362,25 @@ def test_st_navier_stokes_2d():
 
     a = TerminalExpr(-mu*laplace(ue), domain)
     b = TerminalExpr(    grad(ue), domain)
-    c = TerminalExpr(    grad(pe), domain)
+    c = TerminalExpr(    grad(pe), domain)    
     f = (a + b.T*ue + c).simplify()
-
+    
     fx = -mu*(ux.diff(x, 2) + ux.diff(y, 2)) + ux*ux.diff(x) + uy*ux.diff(y) + pe.diff(x)
-    fy = -mu*(uy.diff(x, 2) - uy.diff(y, 2)) + ux*uy.diff(x) + uy*uy.diff(y) + pe.diff(y)
+    fy = -mu*(uy.diff(x, 2) + uy.diff(y, 2)) + ux*uy.diff(x) + uy*uy.diff(y) + pe.diff(y)
 
     assert (f[0]-fx).simplify() == 0
     assert (f[1]-fy).simplify() == 0
-
+    
     f  = Tuple(fx, fy)
     # ...
 
     # Run test
 
     l2_error_u, l2_error_p = run_steady_state_navier_stokes_2d(domain, f, ue, pe, ncells=[2**3,2**3], degree=[2, 2], multiplicity=[2,2])
 
+    print('L2_error_norm(u) = {}'.format(l2_error_u))
+    print('L2_error_norm(p) = {}'.format(l2_error_p))
+
```

Comment:
This will be printed every time the tests run and probably slows them. Maybe we should add a "debug" flag that activates all the prints.

Reply:
sorry I thought that pytest just disregards print statements but it's more complicated than that. I removed these prints which are not needed